src/utils/EntityUtils.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import {
+  Map,
+  get,
+  getIn,
+  isCollection
+} from 'immutable';
+import { Models } from 'lattice';
+
+const { FQN } = Models;
+
+// returns first value of an entity property by FQN
+const getPropertyValue = (
+  entity :Map | Object,
+  fqn :FQN | string,
+) :any => getIn(entity, [fqn, 0]);
```

Comment:
this is already in `lattice-utils`!
https://github.com/openlattice/lattice-utils/blob/develop/src/utils/data/getPropertyValue.js